ci/jenkins/test.sh

```diff
@@ -813,6 +815,12 @@ function run_e2e {
     go mod edit -replace github.com/moby/spdystream=github.com/antoninbas/spdystream@v0.2.1 && go mod tidy
     if [[ $TESTBED_TYPE == "flexible-ipam" ]]; then
         go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus --antrea-ipam
+    elif [[ $TESTBED_TYPE == "kind" ]]; then
+        kubectl config use-context kind-${KIND_CLUSTER}
+        kubectl apply -f build/yamls/antrea-prometheus.yml
+        kubectl apply -f build/yamls/antrea-prometheus-rbac.yml
```

Comment:
why we need to apply the two files in kind testbed only?

Reply:
I think we don't need to apply both files I removed one file.

Reply:
but still why we don't need to apply this for other testbeds?

Reply:
this PR running prometheus tests and we need monitoring and other ns to tests in kind cluster and for that I need to apply prometheus yaml in kind, not sure about other testbeds. @XinShuYang do you have any comment on this?

Reply:
Sorry, Yes right, `deliver_antrea` function runs before running actual tests in `test.sh` script and there we are appending it `cat build/yamls/antrea-prometheus.yml >> build/yamls/antrea.yml ` we don't need to apply prometheus yaml in  `run_e2e` function. thanks!